libs/clj-icat-direct/src/clj_icat_direct/queries.clj

```diff
@@ -517,21 +517,19 @@
 
 (def queries
   {:count-all-items-under-folder
-   "WITH user_groups AS ( SELECT g.*
+   "WITH user_groups AS ( SELECT g.group_user_id
                             FROM r_user_main u
                             JOIN r_user_group g ON g.user_id = u.user_id
                            WHERE u.user_name = ?
                              AND u.zone_name = ? ),
 
-         parent      AS ( SELECT * from r_coll_main
+         parent      AS ( SELECT DISTINCT coll_id, coll_name from r_coll_main
                            WHERE coll_name = ?
-                          UNION
-                          SELECT * from r_coll_main
-                           WHERE coll_name LIKE ? || '/%' ),
+                              OR coll_name LIKE ? || '/%' ),
```

Comment:
Hm, interesting. This could be another collation/indexing for `LIKE` queries issue. When it can use an index on `coll_name` for both conditions those two should be combinable, but I guess the `LIKE` currently can't use any indices.